Client/Frontend/Browser/BrowserViewController/BrowserViewController+KeyCommands.swift

```diff
@@ -6,6 +6,11 @@ import Shared
 
 // Naming functions: use the suffix 'KeyCommand' for an additional level of namespacing (bug 1415830)
 extension BrowserViewController {
+    
+    enum TextSearchDirection: String {
+        case next = "findNext"
```

Comment:
No need to specify rawValue for it, we get it for free

Reply:
Yeah this one is matching the js function to be called for search so either we change case to match or make it this way.

Reply:
oh wait its needed for the `find` method isn't it

Reply:
yeap exactly